sceptre/stack.py

```diff
@@ -136,6 +137,7 @@ def __init__(
             raise InvalidConfigFileError("Neither 'template_path' nor 'template' is set")
 
         self.name = sceptreise_path(name)
+        self.rel_path = rel_path
```

Comment:
The name parameter _is_ the rel_path, just without the ".yaml" on it. I don't really think there's a good reason to accept the rel_path, when the `name` is already available. I think you really should just use the `name` for all the places.